bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/init/NodeInitializers.java

```diff
@@ -97,9 +99,9 @@
     {
         // Default values, one for each instrument type.
         final Map<Instrument, ISPNodeInitializer<ISPObservation, SPObservation>> m =
-                Arrays.asList(Instrument.values()).stream().collect(Collectors.toMap(
+                Arrays.stream(Instrument.values()).collect(Collectors.toMap(
                     Function.identity(),
-                    i -> ObservationNI.forInstrument(i)));
+                        ObservationNI::forInstrument));
```

Comment:
:+1: i can never remember this syntax and just lazily use the lambda form

Reply:
I would probably do the same, but IntelliJ reminds me of it.